### DISCUSSION TITLE
Is there a way to disable motor controls for the Pan V2

### DISCUSSION THREAD

**Discussion by albeedelacruz** [gtxaspec/wz_mini_hacks#132] [Q&A] [answered]
Every reboot kind of messes up the camera's orientation. Probably by disabling motor controls we can set the view to a fixed position just like the v3.

I have 6 of these, so you know how much of a pain it is to constantly adjust the view.

**Comment by gtxaspec** [comment, marked as answer]
good question, let me find out... stay tuned

**Reply by albeedelacruz** [reply]
Thanks!

**Reply by gtxaspec** [reply]
I found the function to hook within libcallback to stop movement.  https://github.com/gtxaspec/wz_mini_hacks/issues/133 

It will be included in a new release soon

**Reply by albeedelacruz** [reply]
This is great! Thank you!

**Reply by albeedelacruz** [reply]
I see motor controls added on the main page. Is there a documentation on how to use it?

**Reply by gtxaspec** [reply]
just enable the variable in the wz mini config file.  set it to true

**Reply by albeedelacruz** [reply]
Hello, I loaded my panv2 with a fresh install of the recent update. Set this value to "true" but I am still able to control the motors from the app. Is there anything else I need to do?

**Reply by albeedelacruz** [reply]
Forgive me, I didn't save the conf. Its working now.

**Reply by gtxaspec** [reply]
got it! great to hear its working for you!

**Reply by albeedelacruz** [reply]
I just want to let you know it is working perfectly well. Previously, a reboot messes up the view finder whenever the motors initialize. Add to the fact that some shared users also accidentally pan the camera when zooming. With the disable motor setting enabled, these are no longer the case. Thanks for all your work!
